pyspark/poetry.lock

```diff
@@ -0,0 +1,1704 @@
+# This file is automatically @generated by Poetry 1.4.2 and should not be changed by hand.
```

Comment:
It seems that this file is temporary generated file，Please double check that why need to include to pyspark module of graphar.

Reply:
It is [highly recommended](https://python-poetry.org/docs/basic-usage/#committing-your-poetrylock-file-to-version-control) to commit this file to git just to have more robust and reproducible builds. I cannot say it that this file is temporary because it contains full list of resolved dependencies. A lot of pupular projects include such a file into VCS: [langchain, 73k stars](https://github.com/langchain-ai/langchain/blob/master/poetry.lock), [Rich, 46k stars](https://github.com/Textualize/rich/blob/master/poetry.lock), etc.

The reason of this file is the same like `npm` lockfile in Node.js, or `cargo` lock file in Rust.

We can drop this file, of course, but this file is one of the core benefits of using `poetry` as a build system.

Reply:
We can remain the file since it's need by poetry, thanks for the explanation.